src/App.js

```diff
@@ -1,13 +1,26 @@
 import React from "react";
-
+import Heading from "./Heading";
 import Bookings from "./Bookings";
-import "./App.css";
+// import "./App.css";
+// import SearchButton from "./SearchButton";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
+// import SearchResults from "./SearchResults.JS";
+
+const data = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789"
+];
```

Comment:
I would try and give this more of an explicit variable name, something like addressData maybe